controllers/storagecluster/exporter.go

```diff
@@ -161,6 +161,12 @@ func getMetricsExporterServiceMonitor(instance *ocsv1.StorageCluster) *monitorin
 func CreateOrUpdateServiceMonitor(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) (*monitoringv1.ServiceMonitor, error) {
 	serviceMonitor := getMetricsExporterServiceMonitor(instance)
 	namespacedName := types.NamespacedName{Name: serviceMonitor.Name, Namespace: serviceMonitor.Namespace}
+	// Adding storagecluster CR name to the metrics as label: managedBy
+	relabelConfig := monitoringv1.RelabelConfig{
+		TargetLabel: "managedBy",
+		Replacement: instance.Name,
+	}
+	serviceMonitor.Spec.Endpoints[0].RelabelConfigs = append(serviceMonitor.Spec.Endpoints[0].RelabelConfigs, &relabelConfig)
```

Comment:
Any reason this isn't in `getMetricsExporterServiceMonitor()`? This struct is static and you already pass `instance` in there anyway.

Reply:
Makes sense. I will fix the PR. Thank you for the review.